src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -82,6 +84,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $helper = new DescriptorHelper($this->fileLinkFormatter);
         $routes = $this->router->getRouteCollection();
         $container = $this->fileLinkFormatter ? \Closure::fromCallable([$this, 'getContainerBuilder']) : null;
+        $sortOption = $input->getOption('sort');
+
+        if (null !== $sortOption) {
```

Comment:
`if ('priority' !== $sortOption) {` (the option cannot be null since it has a default, isn't it?)

Reply:
Right, I removed this check and refactored to:

```
$sortOption = $input->getOption('sort');
$routes = $this->sortRoutes($this->router->getRouteCollection(), $sortOption);
```